examples/platform/silabs/FreeRTOSConfig.h

```diff
@@ -313,7 +310,7 @@ standard names. */
 /* Thread local storage pointers used by the SDK */
 
 #ifndef configNUM_USER_THREAD_LOCAL_STORAGE_POINTERS
-#define configNUM_USER_THREAD_LOCAL_STORAGE_POINTERS 2
+#define configNUM_USER_THREAD_LOCAL_STORAGE_POINTERS 0
```

Comment:
I take it that we didn't need any local storage pointers for thread?

Reply:
From what I can tell no. we don't use it. The slc generated one doesn't set any. I'll be slowly trying to clean up this config and see if we can use the generated one directly